### PR TITLE
Agent: Bring Fixie support back to life

### PR DESCRIPTION
**Pull request by juberti**

### CODE REVIEW

File: packages/voice/src/app/agent/page.tsx [hunk 1]
```diff
@@ -202,7 +202,7 @@ const Button: React.FC<{ onClick: () => void; disabled: boolean; children: React
 const AgentPageComponent: React.FC = () => {
   const searchParams = useSearchParams();
   const agentId = searchParams.get('agent') || 'dr-donut';
-  const agentVoice = getAgent(agentId).ttsVoice;
+  const agentVoice = getAgent(agentId)?.ttsVoice;
```
Comment (petersalas) [inline packages/voice/src/app/agent/page.tsx:205]: I have a couple tweaks incoming to let this work with fixie agent UUIDs...

Comment (juberti) [reply packages/voice/src/app/agent/page.tsx:205]: ok, will sit tight and wait for your PR

Comment (petersalas) [reply packages/voice/src/app/agent/page.tsx:205]: Just pushed it to this branch :P

Comment (petersalas) [reply packages/voice/src/app/agent/page.tsx:205]: Namely, I also tweaked it to ensure that speculative execution is disabled

File: packages/voice/src/app/agent/page.tsx [hunk 2]
```diff
@@ -202,14 +202,14 @@ const Button: React.FC<{ onClick: () => void; disabled: boolean; children: React
...
   const tapOrClick = typeof window != 'undefined' && 'ontouchstart' in window ? 'Tap' : 'Click';
   const idleText = `${tapOrClick} anywhere to start!`;
   const asrProvider = searchParams.get('asr') || DEFAULT_ASR_PROVIDER;
   const asrLanguage = searchParams.get('asrLanguage') || undefined;
   const ttsProvider = searchParams.get('tts') || DEFAULT_TTS_PROVIDER;
   const ttsVoice = searchParams.get('ttsVoice') || agentVoice;
-  const model = searchParams.get('llm') || DEFAULT_LLM;
+  const model = getAgent(agentId) === undefined ? 'fixie' : searchParams.get('llm') || DEFAULT_LLM;
```
Comment (petersalas) [inline packages/voice/src/app/agent/page.tsx:212]: Looks like we took slightly different approaches to identifying things as fixie, but feel free to stomp over my changes however you'd like. Main thing is just to ensure that speculative generation is disabled for fixie

**Pull request merged**